src/utils/dockerDirManager.ts

```diff
@@ -239,11 +236,7 @@ export default class DockerDirManager {
     } else if (platform === 'darwin') {
       return 'osxkeychain';
     } else if (platform === 'linux') {
-      if (currentCredsStore === 'secretservice') {
-        return 'secretservice';
-      } else {
-        return 'pass';
-      }
+      return 'secretservice';
```

Comment:
This has nothing to do with ignoring stdin and seems like a change in functionality.